app/controllers/verify/sessions_controller.rb

```diff
@@ -16,44 +16,39 @@ def new
     end
 
     def create
-      process_step
-    end
-
-    private
+      result = step.submit
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED, result.to_h)
```

Comment:
I saw removed specs re: analytics  from the step class unit specs, but good to know we also have existing specs for this at the controller level! 🌈